### ISSUE TITLE
Monospaced text + Quick texts = Bug?

### ISSUE THREAD

**Issue by Romainrob** [SHWotever/SimHub#492]
Hi.
I may have found a small bug for you to hunt down. Enjoy! ;)

Unless I'm using it wrong, when applying monospacing to (at least some) built in Quick texts, their displayed values seems to stop updating.

Here is an example with Speed, and Cpu Load:
![image](https://user-images.githubusercontent.com/50543913/75093873-2ac8c200-5586-11ea-83dd-3d6488fa7994.png)

[Monospace Test.zip](https://github.com/SHWotever/SimHub/files/4239444/Monospace.Test.zip)

Bon week end!

**Comment by SHWotever** [comment]
fix released in the last beta (b6) ;)
![image](https://user-images.githubusercontent.com/2207331/81438672-6dee0680-916d-11ea-9e40-67fee4679ee0.png)
It was a bit hard, but happy to get it solved, emulating monospacing is a real visual changer !

**Comment by Romainrob** [comment]
Hey! Slow down with the betas. I can barely keep up :)
Nice. Thank you!
When you say "separating each character to set a fixed width comes at a cost"
Are you talking about performance cost?

**Comment by SHWotever** [comment]
Ah ah indeed hard to follow :D I get back to work on Monday so I try to get the final version before :D
Yes it was about performance, each character has to be splitted and centered in its own container to simulate the monospaced effect, it's less efficient than simply drawing a classic text.

**Comment by Romainrob** [comment]
Ok. I'll keep that in mind.
Merci encore!
